### PR TITLE
Adding ReLu function to pytorch frontend 

### PR DESCRIPTION
**Pull request by AymenSe**

### CODE REVIEW

**Comment by AymenSe** [comment]
@chie2727 Sorry, but could you please check my PR because I accidentally deleted the old one which was on the last stage to be merged? @MarShaikh

**Comment by AymenSe** [comment]
@Daniel4078 @ivy-seed Could you please check this PR, it's been 8 days without reviewing.

**Comment by Daniel4078** [comment]
Actually, currently the ivy team member assigned to review this pull request is chie2727 as shown on the assignees column. I will try and contact some related people to deal with this. Thank you for your patience.

**Comment by Daniel4078** [comment]
Well, it seems like chie2727 is not available and I will take over and give you a review soon.

**Review by MarShaikh — CHANGES_REQUESTED** [review]
Just took a look at the PR. Could you please remove the `.idea` files? Other than that, everything else looks good. Please re-request a review from the right hand side after you've made the changes. Thanks.

**Comment by AymenSe** [comment]
relu #2457

**Comment by AymenSe** [comment]
@MarShaikh Could you please take a look ?

**Comment by MarShaikh** [comment]
> @MarShaikh Could you please take a look ?

Could you please resolve the conflict and rerequest for a review. I'll take a look asap. Sorry for the delay!

**Comment by AymenSe** [comment]
> > @MarShaikh Could you please take a look ?
> 
> Could you please resolve the conflict and rerequest for a review. I'll take a look asap. Sorry for the delay!

done! could you please check now?

**Review by MarShaikh — CHANGES_REQUESTED** [review]
Hi, sorry for the delay! There are some linting errors that need to be fixed. Could you look into that? You can always just setup a pre-commit hook, you can find info in our docs. 
Also, if you've only worked on the `relu` function, I would advice to remove the updates to the `sigmoid` function. Thanks!

File: ivy/functional/frontends/torch/non_linear_activation_functions.py [hunk 1]
```diff
@@ -21,7 +28,10 @@ def _compute_elu(input, alpha=1.0, inplace=False):
         return input
     return ret
 
-
+def sigmoid(input, out=None):
```
Comment (MarShaikh) [inline ivy/functional/frontends/torch/non_linear_activation_functions.py:31]: If you're only adding `relu` activation function in the frontend, why have you also added the `sigmoid` function?

**Review by MarShaikh — CHANGES_REQUESTED** [review]

File: ivy/functional/frontends/torch/non_linear_activation_functions.py (hunk 1 above)
Comment (AymenSe) [reply ivy/functional/frontends/torch/non_linear_activation_functions.py:31]: Could you please take a look at it?
actually, Once the conflict occurred, I simply removed the sigmoid function from line 24 and pasted it in line 31.

Comment (MarShaikh) [reply ivy/functional/frontends/torch/non_linear_activation_functions.py:31]: That's why the issue would have occurred! You shouldn't have removed it from the original function and pasted!

**Review by MarShaikh — CHANGES_REQUESTED** [review]
As I said, the issue is still not resolved! Please read the relevant documentation to resolve these issues. Thanks!

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by ivy-seed** [comment]
This PR has been closed because it has been marked as stale for more than 7 days with no activity.

**Pull request closed without merging**